src/map-interactions.react.js

```diff
@@ -325,7 +326,9 @@ export default class Interactions extends Component {
   render() {
     return (
       <div
-        ref="container"
+        ref={container => {
```

Comment:
>Is there a reason to set this.container every render pass?
It's just the way it's done in the react docs, do you have a proposed alternative?

String refs are removed in react@16.0.0-alpha.13 so they will have to be removed eventually regardless...